nengo/tests/test_solvers.py

```diff
@@ -363,7 +362,7 @@ def input_function(t):
 
     for i, Solver in enumerate(Solvers):
         for j, n_neurons in enumerate(neurons):
-            plt.subplot(len(neurons), len(Solvers), len(Solvers)*j + i + 1)
```

Comment:
PEP8 actually says that this should be done the way it originally was here: https://www.python.org/dev/peps/pep-0008/?#other-recommendations

Also, I've noticed that there are a number of stylistic changes in this commit. In my opinion, if they're not violating our static checks (`flake8` and `pylint`), then they should be part of a separate commit. Here, it feels like they make it more difficult to check what is actually being changed.

Reply:
I'm fine reverting those or making them a separate commit, doesn't seem super important to me either way.